File: codeforces/threatresquare.py
class Solution: 
    def ThreatreSquare(self, n: int, m: int, a: int) -> int: 
        resn = (n + a - 1) // a
        resm = (m + a - 1) // a 
        return resn * resm

def main(): 
    n, m, a = map(int, input().split())
    sol = Solution()
    results = sol.ThreatreSquare(n, m, a)
    print(results)
if __name__ == "__main__": 
    main()

File: codeforces/threatresquare.c
#include <stdio.h>

long long Threatresquare(long long m, long long n, long long a)
{
    long long resn, resm; 
    resn = (n + a - 1) / a;
    resm = (m + a - 1) / a;

    return resn * resm;
}

int main()
{
    long long a, m, n; 
    if (scanf("%lld %lld %lld", &n, &m, &a) == 3)
    {
        long long res = Threatresquare(m, n, a);
        printf("%lld \n", res);
    }

    return 0;
    


}
